compiler/main/driver.cpp

```diff
@@ -1392,30 +1433,29 @@ static ArgumentDescription arg_desc[] = {
  {"nil-checks", ' ', NULL, "Enable [disable] runtime nil checking", "n", &fNoNilChecks, "CHPL_NIL_CHECKS", NULL},
  {"stack-checks", ' ', NULL, "Enable [disable] stack overflow checking", "n", &fNoStackChecks, "CHPL_STACK_CHECKS", setStackChecks},
 
- {"", ' ', NULL, "C Code Generation Options", NULL, NULL, NULL, NULL},
+ {"", ' ', NULL, "Code Generation Options", NULL, NULL, NULL, NULL},
  {"codegen", ' ', NULL, "[Don't] Do code generation", "n", &no_codegen, "CHPL_CODEGEN", NULL},
- {"cpp-lines", ' ', NULL, "[Don't] Generate #line annotations", "N", &printCppLineno, "CHPL_CG_CPP_LINES", noteCppLinesSet},
- {"max-c-ident-len", ' ', NULL, "Maximum length of identifiers in generated code, 0 for unlimited", "I", &fMaxCIdentLen, "CHPL_MAX_C_IDENT_LEN", NULL},
  {"munge-user-idents", ' ', NULL, "[Don't] Munge user identifiers to avoid naming conflicts with external code", "N", &fMungeUserIdents, "CHPL_MUNGE_USER_IDENTS"},
  {"savec", ' ', "<directory>", "Save generated C code in directory", "P", &saveCDir, "CHPL_SAVEC_DIR", verifySaveCDir},
-
- {"", ' ', NULL, "C Code Compilation Options", NULL, NULL, NULL, NULL},
```

Comment:
Hmm, confused about this one. This was more specific than what it currently says ("code generation options"), particularly because it distinguishes "generation" from "compilation". Has this change been discussed?

Reply:
The problem here was that this was too specific, and refereed to the C backend when the options also applied to the LLVM backend. I've tried to make it more clear what effects the generation by Chapel vs the compilation by LLVM/GCC/Clang. But I think you are right and something has been lost, let me try something

This has not been discussed, because imo its below the bar for that. All this is doing is adjusting the help output/documentation